
<launch>
    <param name="use_sim_time" value="false"/>

    <!--sliding mode control parameters-->

    <param name="/ac/L_lin" value="5" />
    <param name="/ac/L_ang" value="1" />
    <param name="/ac/Kd_lin" value="5" />
    <param name="/ac/Kd_ang" value="2.5" />
    <param name="/ac/Gamma" value="1500"/>
    <!--<param name="/ac/Gamma" value="25."/>-->
    <param name="/ac/q_filt" value="0.5" />
    <param name="/ac/dq_filt" value="0.9" />
    <param name="/ac/deadband" value="0.3" />
    <param name="/ac/m_init" value="0." />
    <param name="/ac/J_init" value="0" />
    <param name="/ac/wrap_tol" value="1.25" />
    <!--<param name="/ac/a_mags" value="400., 250., 400., 400., 200., 200., 200., 300., 100., 100."/>
    a = [m,J,m*rpx,m*rpy,u1,u1*rix,u1*riy,u1*||ri||,rix,riy]  -->
    <!--<param name="/ac/o_mags" value="20., 100., 20, 20" />
    <param name="ac/g_mags", value="0.001,0.001" />
    <param name="ac/d_mags", value= "0.001, 10, 10, 10," />
    <param name="ac/c_mags", value= "0.001, 10, 10, 0.001" />-->
    <!--<param name="/ac/a_mags" value="10., 100., 5., 5., 2.5, .1, .1, 2., 0.001, 0.001"/>-->

    <param name="/ac/o_mags" value="1e-3, 1e-2, 3e-4, 3e-4" />
    <param name="ac/g_mags" value="0.75e-6, 0.75e-6" />
    <param name="ac/d_mags" value="1.25e-4, 1e-4, 1e-4, 1e-5" />
    <param name="ac/c_mags" value="0.5e-4, 1e-4, 1e-4, 1e-5" />

    <param name="/ouijabot1/offset_angle" value="-1.57" />
    <param name="/ouijabot1/moment_arm" value="0., 0., 0." />
    <param name="/ouijabot2/offset_angle" value="3.14" />
    <param name="/ouijabot2/moment_arm" value="-0.337, -0.215, 0." />
    <param name="/ouijabot3/offset_angle" value="1.57" />
    <param name="/ouijabot3/moment_arm" value="-0.829, -0.209, 0." />
    <param name="/ouijabot4/offset_angle" value="1.57" />
    <param name="/ouijabot4/moment_arm" value="1.174, -0.149, 0." />
    <param name="/ouijabot5/offset_angle" value="-0.52" />
    <param name="/ouijabot4/moment_arm" value="-0.856, 0.441, 0." />
    <param name="/ouijabot6/offset_angle" value="-0.52" />
    <param name="/ouijabot6/moment_arm" value="-0.338, 0.261, 0." />
</launch>
